blog/rust/game-of-life.tsx

```diff
@@ -0,0 +1,215 @@
+//@ts-nocheck
+"use client";
+
+import * as wasm from "@nlb/lrb/game_of_life/game_of_life";
+import { memory } from "@nlb/lrb/game_of_life/game_of_life_bg.wasm";
+import React, { useEffect, useRef, useState } from 'react';
+
+const GameOfLife = () => {
+    const canvasRef = useRef(null);
+    const playButtonRef = useRef(null);
+    const fpsRef = useRef(null);
+    const animationIdRef = useRef<number | null>(null);
+    let lastTimestamp: number | null = null;
+
+    useEffect(() => {
+        console.log("a");
```

Comment:
whoops

Reply:
that's a very import line that lets me know when the component has loaded!